dbt_subprojects/solana/models/_sector/dex/sandwiches/solana/dex_solana_sandwiches.sql

```diff
@@ -29,72 +50,62 @@ WITH indexed_sandwich_trades AS (
+    INNER JOIN dex as victim
+        ON front.block_month=victim.block_month
         AND front.block_time=victim.block_time
         AND front.project_program_id=victim.project_program_id
         AND victim.trader_id BETWEEN front.trader_id AND back.trader_id
         AND front.token_bought_mint_address=victim.token_bought_mint_address
         AND front.token_sold_mint_address=victim.token_sold_mint_address
         {% if is_incremental() %}
         AND {{ incremental_predicate('victim.block_time') }}
-        {% else %}
-        -- AND victim.block_time > NOW() - interval '1' month
         {% endif %}
     CROSS JOIN UNNEST(ARRAY[front.tx_id, back.tx_id]) AS t(tx_id_unwrapped)
-    WHERE front.blockchain='{{blockchain}}'
-    {% if is_incremental() %}
-    AND {{ incremental_predicate('front.block_time') }}
-    {% else %}
-    -- AND front.block_time > NOW() - interval '1' month
-    {% endif %}
-    )
+    WHERE 
+        1 = 1
+        {% if is_incremental() %}
+        AND {{ incremental_predicate('front.block_time') }}
+        {% endif %}
+)
 
 -- Joining back with dex.trades to get the rest of the data & adding block_number and tx_index to the mix
-SELECT dt.blockchain
-, dt.project
-, dt.version
-, dt.block_time
-, dt.block_month
-, tx.block_slot
-, dt.token_sold_mint_address
-, dt.token_bought_mint_address
-, dt.token_sold_vault
-, dt.token_bought_vault
-, dt.token_sold_symbol
-, dt.token_bought_symbol
-, dt.trader_id
-, dt.tx_id
-, dt.token_pair
-, tx.index AS tx_index
-, dt.token_sold_amount_raw
-, dt.token_bought_amount_raw
-, dt.token_sold_amount
-, dt.token_bought_amount
-, dt.amount_usd
-, dt.fee_tier
-, dt.fee_usd
-, dt.project_program_id
-, dt.outer_instruction_index
-, dt.inner_instruction_index
-, dt.trade_source
-FROM {{ ref('dex_solana_trades') }} dt
-INNER JOIN indexed_sandwich_trades s ON dt.block_time=s.block_time
+SELECT
+    dt.blockchain
+    , dt.project
+    , dt.version
+    , dt.block_time
+    , dt.block_month
+    , dt.block_slot
+    , dt.token_sold_mint_address
+    , dt.token_bought_mint_address
+    , dt.token_sold_vault
+    , dt.token_bought_vault
+    , dt.token_sold_symbol
+    , dt.token_bought_symbol
+    , dt.trader_id
+    , dt.tx_id
+    , dt.token_pair
+    , dt.tx_index
+    , dt.token_sold_amount_raw
+    , dt.token_bought_amount_raw
+    , dt.token_sold_amount
+    , dt.token_bought_amount
+    , dt.amount_usd
+    , dt.fee_tier
+    , dt.fee_usd
+    , dt.project_program_id
+    , dt.outer_instruction_index
+    , dt.inner_instruction_index
+    , dt.trade_source
+FROM dex as dt
+INNER JOIN indexed_sandwich_trades as s
+    ON dt.block_month=s.block_month
+    AND dt.block_time=s.block_time
     AND dt.tx_id=s.tx_id
     AND dt.project_program_id=s.project_program_id
--- Adding block_number and tx_index to the mix, can be removed once those are in dex.trades
-INNER JOIN {{source('solana','transactions')}} tx ON tx.block_time=s.block_time
```

Comment:
@andrewhong5297 i was able to remove the join to this raw table, as solana dex has the two fields requested from what i can tell so i simply added in selects